### ISSUE TITLE
Icon fa-eye (regular) different from what is shown on FA website

### ISSUE THREAD

**Issue by Lindorie** [FortAwesome/Font-Awesome#14813]
Hi,

### Describe the problem

The fa-eye icon (regular) I get on my website looks different from what we can see on the official Font Awesome website (here: https://fontawesome.com/icons/eye?style=regular).

![image](https://user-images.githubusercontent.com/9593754/54919943-ebb76a00-4ef9-11e9-900d-7ed4ae341607.png)

As you can see, the pupil is not vertically centered in the eye but is stuck to the top eyelid (like the old FA 4 version) and the white dot in the pupil is going too far into the pupil. 

### What did you expect?

I am expecting the same version as seen on the Font Awesome website:

![image](https://user-images.githubusercontent.com/9593754/54920013-10abdd00-4efa-11e9-923a-33180e372b80.png)

### What version and implementation are you using?

I am using Font Awesome 5 PRO 5.8.1.
I have downloaded the files from our account and I'm loading all.css along with the /webfonts folder. It was doing the same thing with a previous version (not sure which one) so I downloaded the files again and updated everything but it's still showing the wrong icon.

Version: 5.8.1
Browser and version: all

- [ ] SVG with JS
- [X] Web Fonts with CSS
- [ ] SVG Sprites
- [ ] On the Desktop

### Reproducible test case

See here: https://starbit.sedirectoriesdev.co.uk/ the "Visual Search" button 

### Bug report checklist

- [X] I have filled out as much of the above information as I can
- [X] I have included a test case because my odds go _way_ up that the team can fix this when I do
- [X] I have [searched for existing issues](https://github.com/FortAwesome/Font-Awesome/issues) and to the best of my knowledge this is not a duplicate

Thank you

**Comment by tagliala** [comment]
Hi!

Thanks for being part of the Font Awesome Community.

Since you are self-hosting font awesome files, could you please double check that

```
https://starbit.sedirectoriesdev.co.uk/sites/all/themes/seplatform/fonts/fontawesome-pro/webfonts/fa-solid-900.woff2
https://starbit.sedirectoriesdev.co.uk/sites/all/themes/seplatform/fonts/fontawesome-pro/webfonts/fa-regular-400.woff2
https://starbit.sedirectoriesdev.co.uk/sites/all/themes/seplatform/fonts/fontawesome-pro/webfonts/fa-light-300.woff2
```

have been updated to 5.8.1?

I'm pretty confident that `fa-regular-400` **at least** belongs to an older version of Font Awesome (maybe 5.6.3), that would explain why you see `far fa-eye` in that way

**Comment by Lindorie** [comment]
Hi!

Thank you for your quick response! I had updated the webfonts folder and checked it was the latest version but for some reason it was still loading the old files (maybe Drupal cache)...  I had to delete and reupload the updated version of the .woff2 files ; it is now working fine!

![image](https://user-images.githubusercontent.com/9593754/54921922-ec9eca80-4efe-11e9-99a9-82b44ff9c45f.png)

Thank you  :)
